book/src/sec-1-introduction.md

```diff
@@ -1,22 +1,22 @@
 # Introduction
 
-WooriDB is a general purpose (**EXPERIMENTAL**) time serial database, which means it contains all entities registries indexed by DateTime and `Uuid`. It is schemaless, deep key-value storage and uses its own query syntax that is similar to SparQL and Crux's Datalog. 
+WooriDB is a general purpose (**EXPERIMENTAL**) time serial database, which means it contains all entity registries indexed by DateTime and `Uuid`. It is schemaless, deep key-value storage and uses its own query syntax that is similar to SparQL and Crux's Datalog.
 
 ## Name origin
 `Woori` means `our`, in Korean, and although I developed this DB initially alone, it is in my culture and my hunsband's culture to call everything that is done for our communities and by our communities **ours**. I chose *Woori* instead of *shelanu* or *bizdin* because it easier to pronunce.
 
 ## Project inspirations
-- [Crux](https://github.com/juxt/crux) a general purpose bitemporal graph query database with support for SQL and Datalog. It was the ideallogical source of WooriDB. I had developed a rust client called [Transistor](https://github.com/naomijub/transistor) which gave me the basis of what I wanted to have in WooriDB.
-- [Datomic](https://www.datomic.com/) is a transactional database with a flexible data model, temporality and rich queries. I worked with Datomic at [Nubank](https://nubank.com.br/sobre-nos/) and it is the reason I found Juxt/Crux. 
+- [Crux](https://github.com/juxt/crux) a general purpose bitemporal graph query database with support for SQL and Datalog. It was the ideallogical source of WooriDB. I had developed a rust client called [Transistor](https://github.com/naomijub/transistor) which gave me the base of what I wanted to have in WooriDB.
```

Comment:
Both mean the samething in this context, I just prefer `basis` in this context:

"Basis means a starting point, base or foundation for an argument or hypothesis when used as a noun. Bases means foundations or starting points, checkpoints when used as a noun. A good way to remember the difference is Bases is the plural of base. ... Basis is a noun that means "a fundemental principle or a basic unit."

Reply:
Anyhow, rewrote it 43eddd2